servers/rendering/renderer_scene_cull.cpp

```diff
@@ -2239,7 +2239,7 @@ void RendererSceneCull::_light_instance_setup_directional_shadow(int p_shadow_in
 
 			// This trick here is what stabilizes the shadow (make potential jaggies to not move)
 			// at the cost of some wasted resolution. Still, the quality increase is very well worth it.
-			const real_t unit = (radius + soft_shadow_expand) * 2.0 / texture_size;
+			const real_t unit = (radius + soft_shadow_expand) * 4.0 / texture_size;
```

Comment:
I'm a little skeptical about this change. Using a multiplier of 2 makes sense since we are both adding and subtracting ``radius + soft_shadow_expand``. 

I'd expect a visual difference as you are constricting the effective size of the shadowmap. Maybe this should only be change when using 2 splits?

Reply:
there was no visual difference with 4 splits so seems fine to me... though don't ask me why lol